src/plays/sierpinski-triangle/SierpinskiTriangle.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useState } from "react";
+import Sketch from "react-p5";
+import P5 from "p5";
+import { Triangle } from "./Triangle";
+import { getMidPoint } from "./utils";
+import PlayHeader from "common/playlists/PlayHeader";
+
+const SierpinskiTriangle = (props: any) => {
+  const level: number = 6;
+  const multiplier = 0.7;
+  const [windowSize, setWindowSize] = useState([
+    window.innerWidth,
+    window.innerHeight,
+  ]);
+  const [width, height] = windowSize;
+
+  // recursive sierpinski triangle func
+  const drawSierpinskiTriangle = (p: P5, t: Triangle, depth: number) => {
+    // base case
+    if (depth === level) {
+      return;
+    }
+
+    const m1 = getMidPoint(p, t.p1, t.p2);
+    const m2 = getMidPoint(p, t.p2, t.p3);
+    const m3 = getMidPoint(p, t.p3, t.p1);
```

Comment:
Same here give some meaningful names.

Reply:
yes I will look into the naming, thanks for pointing out